dashboard/test/models/blockly_test.rb

```diff
@@ -141,4 +141,84 @@ def assert_equal_xml(a, b)
     level.destroy
     assert_equal xml, xml2
   end
+
+  test 'localizes authored hints' do
+    test_locale = :"te-ST"
+    level_name = 'test_localize_authored_hints'
+
+    I18n.locale = test_locale
+    custom_i18n = {
+      'data' => {
+        'authored_hints' => {
+          "#{level_name}_authored_hint" => {
+            "first": "first test markdown",
+            "second": "second test markdown",
+          }
+        }
+      }
+    }
+
+    I18n.backend.store_translations test_locale, custom_i18n
+
+    level = Level.create(
+      name: level_name,
+      user_id: 0,
+      type: 'Maze',
+      authored_hints: JSON.generate(
+        [
+          {"hint_markdown": "first english markdown", "hint_id": "first"},
+          {"hint_markdown": "second english markdown", "hint_id": "second"},
+        ]
+      )
+    )
+
+    localized_hints = JSON.parse(level.localized_authored_hints)
+
+    assert_equal localized_hints[0]["hint_markdown"], "first test markdown"
+    assert_equal localized_hints[0]["tts_url"], "https://tts.code.org/sharon22k/180/100/1889ea7b2140fc1aef28a2145df32fbb/test_localize_authored_hints.mp3"
```

Comment:
They're generated [here](https://github.com/code-dot-org/code-dot-org/blob/ef8ecf78a204dd2d19a24d9f5a39872ae4a29560/dashboard/app/models/blockly.rb#L334) by the `localized_authored_hints` method, based on the content of the translated markdown